htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -477,7 +477,7 @@ function price2numjs(num)
 
 /* JQuery for product free or predefined select */
 jQuery(document).ready(function() {
-	$("#prod_entry_mode_free").on( "click", function() {
+	$("#tablelines").on( "click", "#prod_entry_mode_free", function() {
```

Comment:
Don t you think we should just replace the on usage by a click usage.
$('#xxx').click(function ...
It looks to be more jquery compliant.